Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Pull base image
+FROM --platform=linux/x86_64 continuumio/miniconda3:23.10.0-1
+
+# Add files
+ADD ./tutorials /home/deeprank2/tutorials 
+ADD ./env/environment.yml /home
```

Comment:
It doesn't make sense to add it in `/home`. `/home` is the directory in linux that contains all the users' home directories. You don't save other data in it. I would put it in /home/deeprank2 where the tutorials are as well. If you want to hide it from the user I would add it in a totally different dir, like `/opt/deeprank2`, or just `/deeprank2`.

Reply:
Indeed I meant to hide them from the user. I edited `/home` with `/deeprank`